spynl/main/mail.py

```diff
@@ -115,7 +115,7 @@ def _sendmail(request, recipient, subject, plain_body, html_body=None,
 
 
 def send_template_email(request, recipient, template_string=None,
-                        template_file=None, plugin=None, replacements={},
+                        template_file=None, replacements={},
                         subject='', fail_silently=True, mailer=None,
                         attachments=None, sender=None, reply_to=None):
     """
```

Comment:
can we also not default to empty dict. default to None and assign the empty dict.